packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -163,9 +165,16 @@ export type HardwareWallet = BaseWallet<
 
 // ----- Bridge Wallet ----- //
 
+export interface BridgeWalletSignInParams extends SignInParams {
+  qrCodeModal?: boolean;
```

Comment:
We need to add the documentation for this new `qrCodeModal` key/property in this file/section:
https://github.com/near/wallet-selector/blob/SQC-288/qr-code-in-ui/packages/core/docs/api/wallet.md#signinparams